query-connector/src/app/query/components/selectQuery/queryHooks.ts

```diff
@@ -85,7 +85,7 @@ export async function fetchQueryResponse(p: {
     p.setIsLoading(true);
     const queryResponse = await UseCaseQuery(
       newRequest,
-      p.queryValueSets.filter((item) => item.include),
+      p.queryValueSets.filter((item) => item.includeValueSet),
```

Comment:
Do we need to do any filtering here at the concept level? Right now, it seems we'd just filter by the upper level valueset, which unless I'm missing something doesn't seem to be granular enough for individual concept-level filtering

Reply:
I _think_ this should be okay as is? Conceptually, if my reasoning for the relationship between `include` and `includeValueSet` is correct, then this should still capture everything that a user has specified at least one concept for. I think this could equivalently be an inclusion of any value set for which at least one member of their `concept` list is set to include.